keda/templates/manager/role.yaml

```
@@ -1,4 +1,4 @@
-{{- if and .Values.certificates.autoGenerated ( not .Values.certificates.certManager.enabled ) }}
+{{- if or (and .Values.certificates.autoGenerated (not .Values.certificates.certManager.enabled)) (and .Values.permissions.operator.restrict.secret .Values.certificates.certManager.enabled) }}
```

If `Values.permissions.operator.restrict.secret` we need this cluster role in any case.
```suggestion
{{- if or (and .Values.certificates.autoGenerated (not .Values.certificates.certManager.enabled)) (.Values.permissions.operator.restrict.secret) }}
```